### PR TITLE
Feat/migrate dh to ghcr

### PR DESCRIPTION
**Pull request by MiroslavRepka**
This PR migrates Claudie's build images from Dockerhub to GHCR.

### CODE REVIEW

**Review by MarioUhrik — APPROVED** [review]
Looks good, thanks Miro..
I don't have time to check in much depth to see if anything's missing, but it looks good :)

File: .github/workflows/CI-pipeline.yml [hunk 1]
```diff
@@ -97,24 +97,23 @@ jobs:
...
-          arr=(${{ needs.check-changes.outputs.ARRAY_OF_CHANGES }})
-          for path in "${arr[@]}"
+          ARR=(${{ needs.check-changes.outputs.ARRAY_OF_CHANGES }})
+          for SERVICE in "${ARR[@]}"
           do
-            echo "-----Building $path-----"
-            IMGTAG="${{secrets.DOCKERHUB_USERNAME}}/$path:${SHORT_GITHUB_SHA}-${GITHUB_RUN_NUMBER}" 
-            DOCKER_BUILDKIT=1 docker buildx build --push --platform linux/amd64,linux/arm64 --tag $IMGTAG -f ./services/$path/Dockerfile . 
+            echo "-----Building $SERVICE-----"
+            IMGTAG="ghcr.io/berops/claudie/$SERVICE:${SHORT_GITHUB_SHA}-${GITHUB_RUN_NUMBER}"
```
Comment (Despire) [inline .github/workflows/CI-pipeline.yml:115]: Just wondering are also going to include the release version in the image tag ?

File: internal/worker/worker.go [hunk 2]
```diff
@@ -39,3 +39,5 @@ func (w *Worker) Run() {
 		}
 	}
 }
+
```
Comment (Despire) [inline internal/worker/worker.go:42]: maybe we should delete this as it suggests 😄

File: .github/workflows/CI-pipeline.yml (hunk 1 above)
Comment (MiroslavRepka) [reply .github/workflows/CI-pipeline.yml:115]: The release version is handled in [release.yml](https://github.com/Berops/claudie/blob/master/.github/workflows/release.yml), more specifically [here](https://github.com/Berops/claudie/blob/feat/migrate-dh-to-ghcr/.github/workflows/release.yml#L65).

File: internal/worker/worker.go (hunk 2 above)
Comment (MiroslavRepka) [reply internal/worker/worker.go:42]: Yeah I will remove it after CI will build images :smile:

**Review by Despire — APPROVED** [review]

**Comment by wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf** [comment]
:tada:

**Pull request merged**